src/classes/StatsD.class.php

```diff
@@ -192,4 +191,67 @@ public function setHost($host)
         $this->host = $host;
     }
 
+    /**
+     * @param string $operation
+     * @param string|array $value
+     * @return array
+     */
+    protected function generateStatData($operation, $value)
+    {
+        $data = array();
+        foreach($this->getStatsPaths() as $path)
+        {
+            $data[$path . ".$operation"]=$value;
+        }
+        return $data;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPivotValue()
+    {
+        return $this->pivotValue;
+    }
+
+    /**
+     * @param string $pivotValue
+     */
+    public function setPivotValue($pivotValue)
```

Comment:
Hmm, interesting question.  As long as `.` isn't the leading or trailing character, I *think* it should be fine.  But some validation is warranted, will do.